.github/workflows/vm-kernel.yaml

```diff
@@ -293,12 +309,26 @@ jobs:
           TAG: ${{ needs.get-tags.outputs.canonical }}
 
         run: |
-          result_image="${VM_KERNEL_IMAGE}:${TAG}"
+          result_image="${GHCR_REGISTRY}/${VM_KERNEL_IMAGE}:${TAG}"
 
           args=("-t" "${result_image}")
           for arch in ${ARCHS}; do
-            args+=("${VM_KERNEL_IMAGE}-${arch}:${TAG}")
+            args+=("${GHCR_REGISTRY}/${VM_KERNEL_IMAGE}-${arch}:${TAG}")
           done
           docker buildx imagetools create "${args[@]}"
 
           echo "image=${result_image}" >> $GITHUB_OUTPUT
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
+
+      - name: Copy vm-kernel to Docker Hub
```

Comment:
nit: do I get it right: that step could be also done with the `_push-to-container-registry`, we just keep it that way for simplicity?

and related question: do I get it right we don't need it in ECR and ACR, right?

Reply:
I don't remember the last time we manually used vm-kernel image, probably we just used it a couple of times when we needed to test specific kernels on staging long time ago. So it should be ok to stop pushing this image to dockerhub.

Note that we still use vm-kernel image for building runner in `build-images.yaml` and that should continue to work.